### DISCUSSION TITLE
Unhandled Exception in CCU Log with HA Integration Enabled

### DISCUSSION THREAD

**Discussion by fr-Fabix** [SukramJ/aiohomematic#1805] [Q&A] [answered]
I’m encountering an unhandled exception in the CCU log whenever the Home Assistant (HA) integration is enabled. Below is a sample of the error log:

`2024-10-30 10:14:28,243 io.vertx.core.impl.ContextImpl ERROR [QEQ0703531_32010_b827ebf2c87c_WorkerPool-0] Unhandled exception 
de.eq3.cbcs.legacy.communication.rpc.RpcRemoteException: Error requesting method event: Cannot read property 'type' of null Code: -1
	at de.eq3.cbcs.legacy.communication.rpc.internal.format.xml.XmlRpcParser.parseMethodResponse(XmlRpcParser.java:154) ~[HMIPServer.jar:?]
	...
	at java.lang.Thread.run(Unknown Source) ~[?:?]`

Steps to Reproduce:

	1.	Enable HA integration with the CCU.
	2.	Check the CCU log for any errors.

When I disable the HA integration entirely, the CCU log is clear, and no errors are generated.

Troubleshooting Steps Taken:

	•	Verified that both CCU and HA integrations are up-to-date. CCU: 3.79.6; HA: 2024.10.4; HA HM Integration: 1.68.1
	•	Disabled all HA programs to rule out issues with individual automations.
	•	Disabled all Homematic entities within HA to isolate the problem.

Expected Behavior:
No errors in the CCU log when the HA integration is enabled.

Actual Behavior:
The CCU log shows continuous unhandled exceptions whenever the HA integration is active.

Additional Information:
It seems that there may be an issue with handling null values in certain property types when processing the event method. Any guidance or recommendations for further troubleshooting would be appreciated.

Any ideas?

**Comment by SukramJ** [comment]
Mit deinen CCU Fehlern solltest Du dich vielleicht besser an eQ-3 oder das Homematic-Forum wenden.

**Reply by fr-Fabix** [reply]
Hi,

eQ-3 ist leider keine Hilfe, sobald man z.B. cuXD einsetzt. Ich denke ich bin schon hier richtig, denn wie beschrieben tritt der Fehler nur auf, wenn die Integration an ist. Aber ich bin gerne auch über andere Hinweise dankbar :-)

**Reply by SukramJ** [reply]
> eQ-3 ist leider keine Hilfe

Deswegen der Hinweis mit dem Homematic-Forum!

>  Ich denke ich bin schon hier richtig

Ich kann mit einer `Unhandled exception` nichts anfangen, und werde auch nichts weiter unternehmen.

**Comment by Baxxy13** [comment, marked as answer]
Solche Fehlermeldungen kann eigentlich niemand außer eQ-3 selbst "deuten" weil die aus den "Closed-Source" - Bestandteilen einer CCUx/RM kommt.
Ich würde mal gucken was `QEQ0703531` für ein Gerät ist und es ggf. mal komplett aus der Zentrale löschen und neu anlernen.

Wenn es nicht zu finden ist könnte es das Funkmodul selbst sein, dazu habe ich dann aber auch keine Idee.
